### DISCUSSION TITLE
The incomplete list of cool Pybricks Projects

### DISCUSSION THREAD

**Discussion by laurensvalk** [pybricks/support#157] [Show and tell]
This is an incomplete collection of awesome creations that users have made with Pybricks so far. Lots of inspiration for your next project! :bulb: :bulb: 

We don't currently have time to maintain a dedicated web page for this, but we don't want your creations to be forgotten either. There's already a few that I could no longer find. So for now, let's collect them here and see how it goes. They're listed in no particular order. Just post a comment below if you'd like your creation to be added, or if you found a project online that's worth sharing.

# Official sets - now with Pybricks
- @albant: [8094 Plotter](https://www.youtube.com/watch?v=gLFPl4FGzL8)
- @jorgepe: [Hedwig the Owl](https://www.youtube.com/watch?v=VQ_j7HJwABw)
- @Patlepirate01: [Hedwig the Owl](https://www.facebook.com/Pybricks/posts/662400101062140)
- @Patlepirate01: [Haunted house](https://www.facebook.com/groups/PoweredUp/permalink/2364060840562040/)
- Pybricks team: [Roller coaster](https://www.facebook.com/Pybricks/posts/632324460736371)
- Pybricks team: [Self balancing 42099 Off Road Truck](https://youtu.be/qi6MRyLSrsI)
- Pybricks team: [EV3 Education Robots](https://pybricks.github.io/ev3-micropython/examples/color_sorter.html)
- @AntoniLuongPham: [EV3 Home Edition Robots](https://github.com/pybricks/pybricks-projects/tree/master/sets/ev3/home)
- @AntoniLuongPham: [EV3 Home Edition Bonus Robots](https://github.com/pybricks/pybricks-projects/tree/master/sets/ev3/home_bonus)

# Bluetooth / remote control
- Dimitri: [SPIKE SwirlBot remote control](https://www.youtube.com/watch?v=l_aQ-F2_25M)
- @jorgepe: [Augmented remote control using a keyboard](https://www.youtube.com/watch?v=esE_6oiFy20)

# Trains:
- Pybricks team: [Automated freight train and switches](https://www.facebook.com/Pybricks/posts/601700160465468)
- Pybricks team: [Train color sorter](https://www.facebook.com/Pybricks/posts/564500934185391)
- Karin: [Automated train barrier](https://www.youtube.com/watch?v=79LWUEo21dg)
- Technicmaster0: [Automated railway crossing](https://www.youtube.com/watch?v=jDTvF2vR95w)

# Machines
- @zphilo: [Ball color sorter](https://youtu.be/YzaEvIgPkdY)
- @kueden: [Time machine](https://www.youtube.com/watch?v=e-JSfwyZt6c)

# Line followers
- @zphilo: [Off road line follower](https://www.youtube.com/watch?v=TFhKRbFzMF0)
- @jorgepe: [Mixed platform line follower](https://www.youtube.com/watch?v=oHwqm5GqPTU)

# Great Ball Contraption
- rjbrickbuilds: [Entry for Great Ball Pit's 2020 GBC Challenge](https://rebrickable.com/mocs/MOC-55141/rjbrickbuilds/rjs-thirteen)

# Games
- @jorgepe: [Google T-Rex game](https://www.youtube.com/watch?v=1HaFRPFWk_I)
- Dimitri: [Mario Piranha Powerslide](https://www.youtube.com/watch?v=JotTs3OWKdU)

# General purpose tools
- @jorgepe: [Hive test](https://www.youtube.com/embed/6-3q4zq-a7w)
- @XilefTech: [CharlieOSX EV3 Coding Tools](https://github.com/XilefTech/CharlieOSX)

**Comment by XilefTech** [comment]
Is this irl-projects connected to a robot of some sort, or just gereally pybricks-coded stuff.

If it is the last one, you can maybe add [CharlieOSX](https://github.com/XilefTech/CharlieOSX)
(might wannt to add a WIP label to it ;) )

I'ts not directly connected to one robot, but more of a universal program that can be used for all kinds of driving robots

**Comment by laurensvalk** [comment]
All Pybricks projects are welcome. I'll add it, thanks!

**Comment by TheVinhLuong102** [comment]
This list is amazing! Love seeing these creations from the PyBricks community!

**Comment by TheVinhLuong102** [comment]
@laurensvalk may I check if PyBricks-API / PyBricks-Projects participate in DigitalOcean's [Hacktoberfest](https://hacktoberfest.digitalocean.com)? Could you make the PyBricks repos Hacktoberfest-participating repos so that we can get Hacktoberfest T-Shirts? :)

**Comment by dlech** [comment]
I added the `hacktoberfest-accepted` label on the PRs from @AntoniLuongPham so hopefully that works.

**Comment by TheVinhLuong102** [comment]
@dlech thank you so much!

**Comment by EclecticInventor** [comment]
I just posted my two arcade games to #1361 and #1362

**Comment by zus2** [comment]
A train / motor controller using the remote that auto detects all motors , lights and a supports 2 hubs 

https://github.com/and-ampersand-and/PyBricks-Train-Motor-Control-Script

**Comment by MrGibbage** [comment]
Can we add a category for FLL Teams? If so, https://github.com/FLL-Team-24277/FLL-Fall-2024-Submerged
